stubs/docutils/METADATA.toml

```diff
@@ -1,4 +1,5 @@
 version = "0.20.*"
+upstream_repository = "https://sourceforge.net/p/docutils/code"
```

Comment:
Oh, could you also add sourceforge to this set here?

https://github.com/python/typeshed/blob/df08fcec5f281ba01266869ec55368819aed0a77/scripts/create_baseline_stubs.py#L112

Reply:
I can. It seems the logic will truncate to `https://sourceforge.net/p/repo`, is that good here? I could special-case sourceforge to add back `/code`

Reply:
Okay, since this is non-trivial in complexity, let's postpone it to a followup PR :)